openml/datasets/functions.py

```diff
@@ -355,7 +355,7 @@ def get_dataset(dataset_id):
 
 def create_dataset(name, description, creator, contributor, collection_date,
                    language, licence, attributes, data, default_target_attribute,
-                   row_id_attribute, ignore_attribute, citation, format="arff",
+                   row_id_attribute, ignore_attribute, citation,
```

Comment:
Same question as before: shall we deprecate the parameters instead of removing it?